src/styles/blocks/page.scss

```
@@ -0,0 +1,6 @@
+.page {
+  margin: 0 auto;
+  padding: 50% 0;
+  display: flex;
+  justify-content: center;
```

you have to set calendar in the middle of the screen => `justify` + `align` supplied with `height` would work good
```suggestion
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
```

---

Thank you for this.